syntax = "proto3";

import "bakdata/corporate/v1/utils.proto";

package bakdata.corporate.v1;

message Announcement {
  string id = 1;
  uint32 rb_id = 2;
  string state = 3;
  string reference_id = 4;
  string event_date = 5;
  string event_type = 6;
  Status status = 7;
  string raw_information = 8;
  string company_name = 10;
  repeated string ceos = 11;
  repeated string shareholders = 12;
}